Src/Support/Google.Apis.Auth/OAuth2/UserCredential.cs

```diff
@@ -49,18 +48,28 @@ public TokenResponse Token
         /// <summary>Gets the user identity.</summary>
         public string UserId { get; }
 
+        /// <summary>
+        /// The ID of the project associated to this credential for the purposes of
+        /// quota calculation and billing.
+        /// </summary>
+        public string QuotaProject { get; }
+
         private readonly TokenRefreshManager _refreshManager;
 
         /// <summary>Constructs a new credential instance.</summary>
         /// <param name="flow">Authorization code flow.</param>
         /// <param name="userId">User identifier.</param>
         /// <param name="token">An initial token for the user.</param>
-        public UserCredential(IAuthorizationCodeFlow flow, string userId, TokenResponse token)
+        /// <param name="quotaProjectId">The ID of the project associated 
+        /// to this credential for the purposes of quota calculation and billing.</param>
+        public UserCredential(IAuthorizationCodeFlow flow, string userId, TokenResponse token, string quotaProjectId = null)
```

Comment:
This is a public constructor, so cannot be changed. Please add a second constructor with the extra parameter.